content/en/BestPractices/defend-against-ssrf.md

```diff
@@ -0,0 +1,257 @@
+# Defend Against Server-Side Request Forgery
+## Description 
+
+[Server-Side Request Forgery (SSRF)](https://owasp.org/Top10/A10_2021-Server-Side_Request_Forgery_%28SSRF%29/) is a server-side security vulnerability that allows a malicious actor (attacker) to make arbitrary requests from the application's server. 
+
+
+SSRF has a wide range of impacts. For example, it could allow an attacker to perform a simple external interaction to steal sensitive AWS metadata information that could eventually lead to an account takeover. SSRF, given its impact and likelihood, has been rated at 10th position in the OWASP Top 10 (2021) vulnerabilities. 
+
+
+
+[Optional] On July 29th, 2019, a widely known organization, CapitolOne, suffered a data breach that exposed 140,000 Social Security Numbers, 80,000 bank account numbers, and 1 million social insurance numbers. The initial attack vector was an SSRF exploit that exposed AWS credential keys. Read more about the attack here. 
+
+
+
+An attacker may achieve the following through a successful SSRF exploitation: 
+
+
+
+- Local/Remote Port Scan
+
+- Local File Read
+
+- Interact with internal apps/service/network
+
+- Remote Code Execution (RCE) by chaining services on the internal network
+
+- Read Metadata Cloud (AWS, Azure, Google Cloud, Digital Ocean, etc.)
+
```

Comment:
For consistency, perhaps `Read Cloud Metadata, such as for Amazon Web Services (AWS), Azure, Google Cloud Platform (GCP), or Digital Ocean` (the "such as" implies the etc.)